patches/chrome-android-java-src-org-chromium-chrome-browser-fullscreen-FullscreenHtmlApiHandlerBase.java.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/chrome/android/java/src/org/chromium/chrome/browser/fullscreen/FullscreenHtmlApiHandlerBase.java b/chrome/android/java/src/org/chromium/chrome/browser/fullscreen/FullscreenHtmlApiHandlerBase.java
+index 65490bf6514e2ab2e8aa28b9c28d6310b82d1341..224c65b657f1c3813402faf0031f26ed2dfb5336 100644
+--- a/chrome/android/java/src/org/chromium/chrome/browser/fullscreen/FullscreenHtmlApiHandlerBase.java
++++ b/chrome/android/java/src/org/chromium/chrome/browser/fullscreen/FullscreenHtmlApiHandlerBase.java
+@@ -113,6 +113,7 @@ public abstract class FullscreenHtmlApiHandlerBase
+     @Nullable private Tab mTab;
+ 
+     private boolean mNotifyOnNextExit;
++    protected boolean mTabHiddenByChangedTabs;
```

Comment:
This can be moved to `BraveFullscreenHtmlApiHandlerBase` and we can access it via `BraveFullscreenHtmlApiHandlerBase.class.cast(this)`

Reply:
Smart solution 👍